ros/src/tl_detector/site_traffic_light_config.yaml

```diff
@@ -8,4 +8,5 @@ stop_line_positions:
 is_site: True
 tl_model_path: /light_classification/tl_models/al_ssd_real/frozen_inference_graph.pb
 tl_detector_rate: 2
```

Comment:
do you think we should have `tl_detection_from_topic:` option in this config as well? I know it will not publish on lot, but at least we could disable GPU if we want to test. Also, code will look for this option when reading config

Reply:
Car lot does not publish the light state. They specifically mentioned it.  No use for it.  Negative.

Reply:
:+1: